db/secondbase/migrate/20200513190850_rename_table_cdr_interval_report_aggrerator.rb

```diff
@@ -0,0 +1,109 @@
+class RenameTableCdrIntervalReportAggrerator < ActiveRecord::Migration[5.2]
+  def change
+    rename_table 'reports.cdr_interval_report_aggrerator', 'cdr_interval_report_aggregator'
+    execute <<-SQL
+      ALTER TABLE ONLY reports.cdr_interval_report_aggregator
+      DROP CONSTRAINT cdr_interval_report_aggrerator_name_key;
+    SQL
+
+    execute <<-SQL
+      ALTER TABLE ONLY reports.cdr_interval_report_aggregator
+      ADD CONSTRAINT cdr_interval_report_aggregator_name_key UNIQUE (name);
+    SQL
```

Comment:
rename instead of dropping & creating

https://www.postgresql.org/docs/11/sql-altertable.html